{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2013-2101",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/access.redhat.com\/security\/cve\/cve-2013-2101",
                    "name": "https:\/\/access.redhat.com\/security\/cve\/cve-2013-2101",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https:\/\/bugzilla.redhat.com\/show_bug.cgi?id=CVE-2013-2101",
                    "name": "https:\/\/bugzilla.redhat.com\/show_bug.cgi?id=CVE-2013-2101",
                    "refsource": "MISC",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "Katello has multiple XSS issues in various entities"
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2019-12-03T14:15Z",
    "lastModifiedDate": "2019-12-03T18:08Z"
}